zh/pause-deployment-of-tidbcluster.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 暂停部署 Kubernetes 上的 TiDB 集群
+summary: 介绍如何暂停部署 Kubernetes 上的 TiDB 集群
+aliases: ['/docs-cn/tidb-in-kubernetes/dev/zh/pause-deployment-of-tidbcluster/']
+---
+
+# 暂停部署 Kubernetes 上的 TiDB 集群
+
+本文介绍如何通过配置文件暂停部署 Kubernetes 上的 TiDB 集群，可用作紧急暂停因人为配置错误或 TiDB Operator 版本兼容性问题导致的 TiDB Cluster 组件滚动升级。
+
+## 部署 Kubernetes 上的 TiDB 集群
+
+参考[快速上手](get-started.md)，在 Kubernetes 上部署 TiDB 集群；或者部署 TiDB 集群到自托管的 Kubernetes 集群[部署到自托管的Kubernetes](deploy-on-general-kubernetes.md)。
+
+## 暂停部署 TiDB 集群
+
+假设管理员发现因人为配置错误或者 TiDB Operator 版本兼容性问题导致了意外的 TiDB Cluster 滚动升级，可以在 TiDB Cluster 配置文件的 `spec` 配置项中新增 `paused: true`，并应用该配置文件，TiDB Cluster 各组件 (PD、TiKV、TiDB、Pump) 的部署过程将会被暂停。配置方式如下所示：
```

Comment:
```suggestion
假设管理员发现因人为配置错误或者 TiDB Operator 版本兼容性问题导致了意外的 TiDB Cluster 滚动升级，可以在 TiDB Cluster 配置文件的 `spec` 配置项中新增 `paused: true`，并应用该配置文件，TiDB 集群各组件 (PD、TiKV、TiDB、Pump) 的部署过程将会被暂停。

1. 修改配置文件如下：
```

Reply:
Please use a numbered list for all the steps in this section.

Reply:
fixed